app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,30 @@
+$Event-color-background: Theme-color(background, default)
```

Comment:
I changed it to `Event-backgroundColor` as you said in the body component. In this event component, how can i be specific without be coupled to the HTML? at the time, The event is a box and the background color for the content. Would `Event-backgroundColor-content` be a good name?